### DISCUSSION TITLE
How to create a custom logstash config?

### DISCUSSION THREAD

**Discussion by ghost** [Security-Onion-Solutions/securityonion#5863] [Unsupported Versions]
Hi, 
I have a security onion 2.80 installed in a virtual box. I have created a custom logstash config file. While this configuration is easily getting loaded in ELK stack, I am having trouble loading this config in security onion. Can you let me know the steps to having the logstash load this custom config and have the logs visualized in Kibana? And also help me to find the users and passwords of these users of elasticsearch. 
Thanks!

**Comment by Sahale** [comment]
_And also help me to find the users and passwords of these users of elasticsearch._
You can find them with `sudo salt-call pillar.get secrets`

**Reply by ghost** [reply]
Thanks for the reply Sahale. 
I was not able to find so_logstash, so_elastic, or any of the other passwords of superusers of elasticsearch. Can you help me out?

**Comment by ghost** [comment]
Any updates on this topic?
